code/centauri/ibc/modules/ics03_connection.md

```diff
@@ -0,0 +1,105 @@
+## ICS03_CONNECTION
+
+IBC connections are direct links between chains, it is recommended to have only one open connection between two  
+specific chains, which translates to one connection per light client.  
```

Comment:
for cost reasons?

Reply:
No, it's just unnecessary to have multiple connections can be confusing to users.